fineract-provider/dependencies.gradle

```diff
@@ -66,7 +66,7 @@ dependencies {
 
                 //[group: 'mysql', name: 'mysql-connector-java', version: '5.1.27'],
                 [group: 'org.drizzle.jdbc', name: 'drizzle-jdbc', version: '1.4'],
-               // [group: 'org.apache.tomcat', name: 'tomcat-jdbc', version: tomcatVersion],
+                [group: 'org.apache.tomcat', name: 'tomcat-jdbc', version: tomcatVersion],
```

Comment:
New version of spring-boot-starter-data-jpa uses HikariCP as default connection pool instead of tomcat-jdbc. So I need to add this dependency. Also I have excluded HikariCP in the latest commit.

Reply:
OK, thanks for the clarification. I've opened https://issues.apache.org/jira/browse/FINERACT-796 suggesting that someone (you?) could replace Tomcat JDBC with HikariCP later.